### ISSUE TITLE
Integration tests are no longer executed

### ISSUE THREAD

**Issue by Zelldon** [camunda/camunda-platform-helm#492]
It looks like the integration tests are no longer executed.

![test](https://user-images.githubusercontent.com/2758593/203230338-b5681864-4787-4163-bb86-cf2bdd62f16d.png)

![itest](https://user-images.githubusercontent.com/2758593/203230586-9b9c4b8e-21a5-4d20-b018-761952f635d5.png)

Last change on `.github` was removing my name  from the Codeowners file idk whether this is related

![change](https://user-images.githubusercontent.com/2758593/203230162-5f06afb5-6572-46b8-874e-041170b674c8.png)

**Comment by aabouzaid** [comment]
No, your change is not related at all.
The integration tests were super flaky and could not run unattended (always something fails).

In #475, I've enhanced a bit to run each test case individually and allow rerunning of the failed ones only (before, all tests ran in the same github action build, and if one failed, we needed to rerun everything and we needed a couple of runs to have all of them green at the same time).

I've also limited them to the [PRs only](https://github.com/camunda/camunda-platform-helm/blob/651fa9368a7e864a7fe581b9a5a0350d7aa01cd9/.github/workflows/test-integration.yaml#L4) since now we have a PR-based release so they will always run with new features and new releases.

We have a plan to review the integration tests to find the best way to have them stable and with a short feedback loop.
